internal/cri/server/container_create.go

```diff
@@ -462,6 +462,9 @@ func (c *criService) createContainer(r *createContainerRequest) (_ string, retEr
 	if err := c.containerStore.Add(container); err != nil {
 		return "", fmt.Errorf("failed to add container %q into store: %w", r.containerID, err)
 	}
+	if c.statsCollector != nil {
```

Comment:
same here.. over in the container store is best given some actions, restart etc.. don't involve a CRI call

Reply:
Sorry i lost you, what should i do here?

Reply:
ack!

Reply:
@mikebrow hmm now i remember, c.statsCollector is not available there in the Create method and i did not want to pass that along!